### DISCUSSION TITLE
How can I disable readonly for Match?

### DISCUSSION THREAD

**Discussion by justin-pierce** [fastlane/fastlane#20228] [Q&A] [answered]
I guess readonly was enabled during setup. Any way to change that setting so it can generate new profiles?

**Comment by lucgrabowski** [comment]
I don't know if you use it with Fastfile or from command line, so take a look at the docs https://docs.fastlane.tools/actions/match/ you'll see how to use both of those methods.

**Comment by justin-pierce** [comment, marked as answer]
Set it up with CLI and use it with fastlane, but to answer my question: I just used CLI to regenerate the profile to get it working again.
